runregistry-rest/rest.py

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env python3
-"""
+'''
```

Comment:
Any reason to switch these?

Reply:
Also remenants of the merge with Pierres PR